tests/qos/test_qos_sai.py

```diff
@@ -1892,15 +1907,32 @@ def testQosSaiQWatermarkAllPorts(
         allTestPortIps = []
```

Comment:
need to initialize "testPortIps" here, 
e.g.
```
testPortIps = dutConfig["testPortIps"]
```
otherwise, will get failure when execute 
```
             allTestPortIps = [
                 testPortIps[get_src_dst_asic_and_duts['dst_dut_index']]
```

Reply:
@XuChen-MSFT : "testPortIps" already initiated in the 2nd line of the testcase. Does that address your concern ?